### DISCUSSION TITLE
code behind can't see controls on main page

### DISCUSSION THREAD

**Discussion by stevenroman760** [unoplatform/uno#8347] [Q&A]
In my mainpage.xaml in my shared project, the code behind in mainpage.xaml.cs can't see the textbox controls. What am I missing? In case it matters I also can't use the designer and the rebuild reload fix I saw on the forum didn't help.

**Comment by jeromelaban** [comment]
Thanks, make sure to follow the exact steps, as well as selecting the UWP platform on the top left selector as mentioned here: https://platform.uno/docs/articles/get-started-wizard.html

**Comment by sasakrsmanovic** [comment]
As discussed on the phone, a few steps to confirm is 1 - go through the tutorial here which will help validate your setup is right https://platform.uno/docs/articles/getting-started-tutorial-1.html

In addition, please try VS 2022 as you are using VS 2019 now. 

In case you continue to have challenges after these two steps, please write back and send some snips and screenshots, as well as exact Visual Studio edition (including any 'preview' remarks etc - from your "About Visual Studio"  screen.

**Reply by stevenroman760** [reply]
Installing Visual Studio 2022 resolved my issue. Thank you!
